lib/data/use_case/anime/get_top_anime.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../../presentation/feature/news/models/anime_news_item_model.dart';
+import '../../model/api/request/get_top_anime_request.dart';
+import '../../providers/my_anime_list_repository_provider.dart';
+
+part 'get_top_anime.g.dart';
+
+@riverpod
+Future<List<AnimeNewsItemModel>> topAnime(
```

Comment:
```suggestion
Future<List<AnimeNewsItemModel>> getTopAnimeUseCase(
```
I think we should make the file with the same name?

Reply:
This is only an opinion, for the template I think the anime folder is not necessary, if not the folder tree is too big and is difficult to understand